connectors/testStandaloneCosmetic/src/test/scala/io/delta/standalone/internal/ShadedJarSuite.scala

```diff
@@ -70,7 +70,7 @@ class ShadedJarSuite extends FunSuite {
   test("basic read and write to verify the final delta-standalone jar is working") {
     val dir = Files.createTempDirectory(UUID.randomUUID().toString).toFile
     try {
-      val tablePath = new File("../golden-tables/src/test/resources/golden/data-reader-primitives")
+      val tablePath = new File("../golden-tables/src/main/resources/golden/data-reader-primitives")
```

Comment:
can we use the `getResource` to find the path?

Reply:
done!